packages/social-previews/src/twitter-preview/tweet.jsx

```diff
@@ -94,7 +94,7 @@ export class Tweet extends PureComponent {
 	 */
 	renderMedia( media ) {
 		if ( ! media ) {
-			return;
+			return null;
```

Comment:
Should we include `| null` in the returned type above?

Reply:
I think React allows null as an element, so IMO we don't need to (since this is a react component after all)